packages/solid/components/Keyboard/KeyboardSimple.tsx

```diff
@@ -153,7 +153,7 @@ const KeyboardSimple: Component<KeyboardProps> = (props: KeyboardProps) => {
                       display={'flex'}
                       gap={gap()}
                       height={keyHeight()}
-                      wrap={props.rowWrap}
+                      wrap={props.rowWrap as boolean}
```

Comment:
instead of asserting a type here we should add rowWrap to the key props or keyboard props, wherever it belongs